packages/docs/.vitepress/theme/index.ts

```
@@ -1,16 +1,16 @@
-import TipAlert                                       from '@/.vitepress/theme/Components/ComponentDocs/UI/TipAlert.vue';
-import Layout                                         from '@/.vitepress/theme/Components/Layout/UI/Layout.vue';
-import CodePreview                                    from '@/.vitepress/theme/Components/Preview/UI/CodePreview.vue';
-import hljsVuePlugin                                  from '@highlightjs/vue-plugin';
-import { FoKeyboard, FoSelectThemeController, vMask } from 'flyonui-vue';
-import { createPinia }                                from 'pinia';
-import DefaultTheme                                   from 'vitepress/theme';
-import { defineAsyncComponent }                       from 'vue';
+import TipAlert                                                            from '@/.vitepress/theme/Components/ComponentDocs/UI/TipAlert.vue';
+import Layout                                                              from '@/.vitepress/theme/Components/Layout/UI/Layout.vue';
+import CodePreview                                                         from '@/.vitepress/theme/Components/Preview/UI/CodePreview.vue';
+import hljsVuePlugin                                                       from '@highlightjs/vue-plugin';
```

ESLint

[error] Expected "@highlightjs/vue-plugin" (external) to come before "@/.vitepress/theme/Components/Preview/UI/CodePreview.vue" (internal).

(perfectionist/sort-imports)